src/test/java/org/dependencytrack/parser/snyk/SnykParserTest.java

```diff
@@ -93,21 +94,21 @@ public void testParseSeveritiesNvd() throws IOException {
 
         // By default NVD is first priority for CVSS, no need to set config property.
```

Comment:
Are you asking in general, or specific for Snyk?

Generally we do not re-use scoring of aliasing vulnerabilities, if a given vuln does not have a scoring.

For Snyk specifically, they provide at least one score for each vulnerability. Beside Snyk's own scoring, and the "offical" NVD one, there is also vendor-specific scoring from Red Hat, SUSE, etc.; In that case DT should follow a priority list of sorts, where it would fall back to the next preferred source if necessary.

Reply:
yes thats what i was thinking.  following a sequence of logic if there happens to be a vuln that does not have a score (for some reason) to default to the next best source.